portfolio/src/main/webapp/script.js

```diff
@@ -92,3 +92,11 @@ window.addEventListener('scroll', function() {
     document.getElementById('logo').style.fontSize = '35px';
   }
 })
+
+/**
```

Comment:
Can you add some comments for this function here?

Reply:
Oh, wow, forgot to add it...It's done now!